x-pack/plugins/fleet/public/components/cloud_formation_guide.tsx

```diff
@@ -44,12 +50,21 @@ export const CloudFormationGuide = () => {
       </p>
       <EuiText size="s" color="subdued">
         <ol>
-          <li>
-            <FormattedMessage
-              id="xpack.fleet.cloudFormation.guide.steps.login"
-              defaultMessage="Ensure you are logged in as an admin in the AWS Account you want to onboard"
-            />
-          </li>
+          {awsAccountType === 'organization-account' ? (
+            <li>
+              <FormattedMessage
+                id="xpack.fleet.cloudFormation.guide.steps.organizationLogin"
+                defaultMessage="Log in as an admin in the management account of the AWS Organization you want to onboard"
```

Comment:
any specific reason to have different wording here and in `aws_credentials_form`?

Reply:
I'm not sure, one is before the "save and continue" button is pressed, the others are for the modals and flyouts. 

@tinnytintin10 @einatjacoby any particular reason for not keeping the texts the same? or should we open a follow up pr?

Reply:
I was just confused by the fact that for the Cloudformation option, two texts are identical `Log in as an admin in the AWS account you want to onboard` and `Log in as an admin to the AWS Account you want to onboard` (except the `Account` starting from a capital letter)  and for the AWS Org texts differ but the meaning seems the same for me.
Not a big deal, I guess I just like consistency :)

Reply:
I'm 100% with you, I assume it was a mistake since those texts were changed last minute since the figma had some inaccuracies. I'll open a pr and will make sure its ok by product before merge.